src/index.html

```diff
@@ -4,10 +4,80 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Stars</title>
+    <title>!T!Stars</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      rel="shortcut icon"
+      href="images/star-active.svg"
+      type="image/x-icon"
+      >
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars__0">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars__1">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars__2">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars__3">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars__4">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars__5">
```

Comment:
Modifier should be indicated by -- in BEM
```suggestion
    <div class="stars stars--5">
```